build-system/tasks/presubmit-checks.js

```diff
@@ -1184,14 +1184,10 @@ const forbiddenTermsSrcInclusive = {
     message: 'Unsupported on IE; use trim() or a helper instead.',
     allowlist: ['validator/js/engine/validator.js'],
   },
-  "process\\.env(\\.TRAVIS|\\[\\'TRAVIS)": {
+  "process\\.env(\\.|\\[\\')(TRAVIS|GITHUB_ACTIONS|CIRCLECI)": {
```

Comment:
```suggestion
  /process\.env(\.|\[['"])(TRAVIS|GITHUB_ACTIONS|CIRCLECI)/: {
```

Reply:
This follows the conventions set by this file. (Side note: It's a longer-term goal to eliminate regex based presbubmits for something more powerful / reliable.)